src/components/text/presets/textPresets.ts

```diff
@@ -1,8 +1,8 @@
-export { default as CdrHeadingDisplay } from './PresetHeadingDisplay.vue';
-export { default as CdrHeadingSerif } from './PresetHeadingSerif.vue';
-export { default as CdrHeadingSans } from './PresetHeadingSans.vue';
-export { default as CdrSubheadingSans } from './PresetSubheadingSans.vue';
-export { default as CdrUtilitySans } from './PresetUtilitySans.vue';
-export { default as CdrUtilitySerif } from './PresetUtilitySerif.vue';
-export { default as CdrBody } from './PresetBody.vue';
-export { default as CdrEyebrow } from './PresetEyebrow.vue';
\ No newline at end of file
+export { default as CdrHeadingDisplay } from './CdrHeadingDisplay.vue';
+export { default as CdrHeadingSerif } from './CdrHeadingSerif.vue';
+export { default as CdrHeadingSans } from './CdrHeadingSans.vue';
+export { default as CdrSubheadingSans } from './CdrSubheadingSans.vue';
+export { default as CdrUtilitySans } from './CdrUtilitySans.vue';
+export { default as CdrUtilitySerif } from './CdrUtilitySerif.vue';
+export { default as CdrBody } from './CdrBody.vue';
+export { default as CdrEyebrow } from './CdrEyebrow.vue';
```

Comment:
Vite was naming the CSS files according to the native component name so dropping this naming convention